packages/components/src/Disclosure/docs/controlled.tsx

```diff
@@ -0,0 +1,24 @@
+import { Disclosure, DisclosureHeader, DisclosurePanel, Div } from "@hopper-ui/components";
+import { useState } from "react";
+
+export default function Example() {
+    const [isExpanded, setIsExpanded] = useState(true);
+
+    return (
+        <Div width="100%" paddingX="core_320" paddingY="core_480">
```

Comment:
why are all the stories wrapped with this. It prevent me from knowing what the component looks like

Reply:
how does it prevent you from knowing what it looks like?
The issue is the styling for our examples. It has display flex, which will shrink everything in the middle. 
For disclosure, this means it starts off small, and once you open it, it'll expand in width due to the Panel having more content.
This seems like a horrible user experience to me.
Also, I add the padding do that the disclosure doesn't show under the buttons in the example box.